src/lib/messenger.js

```diff
@@ -23,6 +23,19 @@ export default function messenger(bot) {
       const text = message.text.toLowerCase();
       const user = message.user || {};
 
+      function stripText(input) {
```

Comment:
maybe a little zealous, but when reading the function name `stripText` it indicated that the text itself is stripped, not the command :)